hackathon_site/event/jinja2/event/reset_password/reset_password.html

```diff
@@ -1,31 +1,37 @@
 {% extends "event/form_base.html" %}
 
 {% block form_title %}Password Reset{% endblock %}
-{% block form_description %}<p>Forgot your password? Enter your email address below, and we’ll email instructions for setting a new one.</p>{% endblock %}
+
+{% block nav_logout_login_links %}{% endblock %}
+
+{% block form_description %}<p class="col s12">Forgot your password? Enter your email address below and we’ll email instructions for setting a new one.</p><br />{% endblock %}
```

Comment:
`form_description` is already inside a `col s12` class div, so you have a `col s12` inside another one now without an intermediary row

Reply:
my brain hurts rn and I will look at this tmr